### DISCUSSION TITLE
How to get Query String/ HTTP request in configure service method in Asp.net Core

### DISCUSSION THREAD

**Discussion by Raja-Ashutosh** [dotnet/aspnetcore#22502] [General]
How to get Query String/ HTTP request in configuring service method in Asp.net Core

**Comment by mkArtakMSFT** [comment]
Hi. It looks like this is a question about how to use ASP.NET Core. While we do our best to look through all the issues filed here, we are not a general-purpose forum. To get a faster response we suggest posting your questions to [StackOverflow](https://www.stackoverflow.com) using the **asp.net-core-mvc** tag.

**Comment by Tratcher** [comment]
Startup.ConfigureServices runs before the server starts accepting requests, it's not possible to get access to any Http requests. Why do you want to?
